lib/hackerone/client/weakness.rb

```diff
@@ -0,0 +1,43 @@
+module HackerOne
+  module Client
+    class Weakness
+      class << self
+        def extract_cwe_number(cwe)
+          fail StandardError::ArgumentError unless cwe.upcase.start_with?('CWE-')
+
+          cwe[4..-1].to_i
```

Comment:
I'd prefer `split`ing the string and parsing out the two pieces of the value over magic numbers.